### PR TITLE
Func tests/add tests for updating bw

### PR DESCRIPTION
**Pull request by ylobankov**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy [hunk 1]
```diff
@@ -38,15 +37,21 @@ class LinkPropertiesSpec extends BaseSpecification {
...
+    def "Link props are created with empty values when sending not a valid link props key"() {
         when: "Send link property request with invalid character"
         def response = northbound.updateLinkProps([new LinkPropsDto("00:00:00:00:00:00:00:01", 1,
-                "00:00:00:00:00:00:00:02", 1, ['`cost': "700"])])
+                "00:00:00:00:00:00:00:02", 1, ["`cost": "700"])])
 
-        then: "Response states that operation failed"
-        response.failures == 1
-        response.messages.first() == "Invalid request"
+        then: "Response states that operation succeeded"
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy]: I find this to be a defect and not an expected behavior. This was introduced during pre-H&S release and was considered a minor issue (low priority)

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy [hunk 2]
```diff
@@ -128,30 +141,40 @@ class LinkPropertiesSpec extends BaseSpecification {
...
+    def "Updating cost and max bandwidth via link props actually updates cost and max bandwidth on ISLs"() {
+        given: "An active ISL"
         def isl = topology.islsForActiveSwitches.first()
+        def initialMaxBandwidth = islUtils.getIslInfo(isl).get().maxBandwidth
 
-        when: "Update cost on ISL via link props"
-        def cost = "12345"
-        def linkProps = [islUtils.toLinkProps(isl, ["cost": cost])]
+        when: "Update cost and max bandwidth on ISL via link props"
+        def value = "12345"
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy]: I would've gone for different values for cost/bw to verify that they do not get mixed up during update

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy [hunk 3]
```diff
@@ -496,24 +499,24 @@ class LinkSpec extends BaseSpecification {
...
-    def "Isl is able to properly fail when both src and dst switches suddenly disconnect"() {
+    def "Link is able to properly fail when both src and dst switches suddenly disconnect"() {
         requireProfiles("virtual")
 
-        given: "An ISL under test"
+        given: "An active ISL"
         def isl = topology.islsForActiveSwitches.first()
 
-        when: "Source and destination switches of the ISL suddenly disconnect"
+        when: "Source and destination switches of the link suddenly disconnect"
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy]: I doesn't really matter, but I don't fully agree with these renames. If you consider `link` to be a bi-directional entity, then it shouldn't have neither `source` nor `destination`.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy (hunk 2 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy]: Fixed.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy (hunk 3 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy]: Fixed.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy (hunk 1 above)
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy]: Yes, I understand but I would like to verify that Kilda is not hanging as it was before. I will add a TODO with mention that it is a defect and in the future the test should be aligned with the right behavior.

**Pull request merged**